misc/ref/example_seismic.py.out

```diff
@@ -0,0 +1,3 @@
+pressure is not defined for IASP91
+gravity is not defined for IASP91
+density is not defined for IASP91
```

Comment:
When I run the script I also get warning output about pressure and gravity for STW105 and AK135